src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -396,6 +382,10 @@ The router will trigger in-page transitions from `<form>` elements, supporting b
 </form>
 ```
 
+:::tip
+By default your page will reload after the `POST` request is sent, if you are using Javascript to prevent this with `event.preventDefault()` you will also want to opt out of router transitions for this form. 
+:::
```

Comment:
What's the alternative? What happens when `event.preventDefault()` is used without opting out?

Reply:
ViewTransitions *transitions* to the default (either sends post or get) which if user is not set up for that seems like it just refreshes the page